cdn_lambda/functions/origin_response/origin_response.py

```diff
@@ -37,23 +37,24 @@ def handler(self, event, context):
                 "value"
             ]
 
-        except (KeyError):
+        except KeyError:
             LOG.debug(
-                "Could not read exodus-original-uri from response", exc_info=1
+                "Could not read exodus-original-uri from response",
+                exc_info=True,
             )
             original_uri = None
 
-        if original_uri is not None:
+        if original_uri:
             for pattern in max_age_pattern_whitelist:
                 if re.match(pattern, original_uri):
                     response["headers"]["cache-control"] = [
                         {
                             "key": "Cache-Control",
-                            "value": "max-age={}".format(max_age),
+                            "value": f"max-age={max_age}",
                         }
                     ]
                     LOG.info(
-                        "Cache-Control header added for '%s'", original_uri
+                        f"Cache-Control header added for '{original_uri}'"
```

Comment:
Pylint complains about this and I agree, we should still let the logger take care of interpolation.  f-strings doesn't seem to change this.

This is the reason for CI failure.